doc/configuration.md

```diff
@@ -21,7 +21,7 @@ Parameters controlling the Cluster-API management server (capi management server
 | `ssh_username`           |                 | SCS        | `ubuntu`          | Name of the default user for the `image`                                      |
 | `clusterapi_version`     |                 | SCS        | `1.3.5`           | Version of the cluster-API incl. `clusterctl`                                 |
 | `capi_openstack_version` |                 | SCS        | `0.7.1`           | Version of the cluster-api-provider-openstack (needs to fit the CAPI version) |
-| `cilium_binaries`        |                 | SCS        | `v0.13.1;v0.11.2` | Versions of the cilium and hubble CLI in the vA.B.C;vX.Y.Z format             |
+| `cilium_binaries`        |                 | SCS        | `v0.13.3;v0.11.6` | Versions of the cilium and hubble CLI in the vA.B.C;vX.Y.Z format             |
```

Comment:
```suggestion
| `cilium_binaries`        |                 | SCS        | `v0.13.2;v0.11.6` | Versions of the cilium and hubble CLI in the vA.B.C;vX.Y.Z format             |
```

Reply:
Thanks for spotting this, @chess-knight !
Rather than accepting your 3 individual suggestions via github (resulting in 3 commits on the branch), I created one commit that fixes all three places.